rel-eng/tito.props

```diff
@@ -133,6 +133,7 @@ whitelist = foreman
   rubygem-gridster-rails
   rubygem-hammer_cli
   rubygem-hammer_cli_foreman
+  rubygem-foreman_maintain
```

Comment:
This one is fine, but I am not seeing an entry under the `foreman-nightly-nonscl-rhel7` section.